assets/css/_page.scss

```diff
@@ -0,0 +1,3 @@
+.c-page {
+  margin-left: $tab-bar-width;
```

Comment:
Instead of positioning everything absolutely, and getting them to fit around each other, would it be better to postition these both as part of the same layout flow, and put them next to each other? There are a lot of things that need to know about `$tab-bar-width` that feel unrelated.

Reply:
I did try that initially, but ran into problems when trying to get the RouteLadder/TabBar collapse working correctly while also having the TabBar appear _without_ the RouteLadder on the FAQ page. I agree that it's not ideal to have multiple things depend on `$tab-bar-width`, but this was the best solution i found that worked.